shared.go

```diff
@@ -40,6 +44,27 @@ func (a App) Lower() string {
 	return strings.ToLower(string(a))
 }
 
+// Client returns the default client, and is used if one is not passed in.
+func Client(timeout time.Duration, verifySSL bool) *http.Client {
+	return &http.Client{
+		Timeout: timeout,
+		CheckRedirect: func(r *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: !verifySSL}, //nolint:gosec
+		},
+	}
+}
+
+// ClientWithDebug returns an http client with a debug logger enabled.
+func ClientWithDebug(timeout time.Duration, verifySSL bool, logConfig debuglog.Config) *http.Client {
```

Comment:
Where are you using this one?

Reply:
notifiarr, it's still in a branch, but it's going out soon. The idea of this method was keep feature parity with what this library already provided.

Reply:
And if it wasn't clear, you can use this method with non-start apps. You can use it to create a debug http client for pretty much any app/library.